addons/parachute/XEH_postInit.sqf

```diff
@@ -33,7 +33,7 @@ GVAR(PFH) = false;
 ["vehicle",{
     if (!GVAR(PFH) && {(vehicle ACE_player) isKindOf "ParachuteBase"}) then {
         GVAR(PFH) = true;
-        [FUNC(onEachFrame), 0.1, []] call CALLSTACK(CBA_fnc_addPerFrameHandler);
+        [FUNC(onEachFrame), 0.1, []] call CBA_fnc_addPerFrameHandler;
     };
 }] call CBA_fnc_addPlayerEventHandler;
```

Comment:
The whole thing can be deleted I think.

Reply:
It needs to run to determine when the player lands to reset the chute cut variable. Which is what I mention in the OP about if we change the way those variables are handled we can totally remove the PFH.

Reply:
(At least, I'm assuming that variable being reset is necessary. It's quite hard to determine what the original intent is with some of the code)